Src/Sarif.PatternMatcher.Cli/StressCommand.cs

```diff
@@ -286,8 +365,11 @@ private void TimeScanFileWithSkimmers(string filePath, ISet<Skimmer<AnalyzeConte
                 logger.AnalysisStopped(RuntimeConditions.None);
             }
 
+            TimeSpan endCpu = Process.GetCurrentProcess().TotalProcessorTime;
             timer.Stop();
-            fileDataTupleList.Add(Tuple.Create(filePath.Replace(',', ';'), fileSystem.FileInfoLength(filePath) / 1024, timer.ElapsedMilliseconds));
+
+            double cpuUtilization = 100 * (endCpu.TotalMilliseconds - startCpu.TotalMilliseconds) / (Environment.ProcessorCount * timer.ElapsedMilliseconds);
```

Comment:
>cpuUtilization [](http://example.com/codeflow?start=19&length=14)

nit: create a function for the cpuUtilization for reusing

Reply:
@yongyan-gh good idea. done.